zerver/openapi/zulip.yaml

```
@@ -9225,6 +9225,24 @@ paths:
                           array will be empty if `enable_drafts_synchronization` is set to `false`.
                         items:
                           $ref: "#/components/schemas/Draft"
+                      emoji_data_url:
+                        type: string
+                        description: |
+                          The URL to a JSON file that describes which emoji names map to which
+                          emoji codes, for all Unicode emoji this Zulip server accepts. The data
+                          at the given URL is a JSON object with one property, `name_to_code`.
```

Paragraph break here:

```suggestion
                          emoji codes, for all Unicode emoji this Zulip server accepts.

                          The data
                          at the given URL is a JSON object with one property, `name_to_code`.
```

Also before "The data at this URL is immutable", making a third paragraph. (Though based on the discussion up to here:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/currently.20supported.20emoji/near/1397835
the current plan is we'll actually just delete that third paragraph.)